CMSIS/NN/Source/ActivationFunctions/arm_relu_q7.c

```diff
@@ -66,15 +66,15 @@ void arm_relu_q7(q7_t *data, uint16_t size)
 
     while (i)
     {
-        in = read_q7x4_ia(&input);
+        in = arm_nn_read_q7x4_ia((const q7_t **)&input);
```

Comment:
Why is input not defined as const ?

Reply:
It is one of the pre-TFLM API's where the output location is same as the input.

Most operators now have the activation function fused and this is not used.